background/get-userscripts.js

```diff
@@ -32,12 +32,11 @@ async function getContentScriptInfo(url) {
     addonsWithUserscripts: [],
     userstyleUrls: [],
     themes: [],
+    allAddons: [],
```

Comment:
There's no need for `allAddons` unless you're implementing userscript autorefresh.